whipper/image/toc.py

```diff
@@ -158,33 +158,29 @@ def _index(self, currentTrack, i, absoluteOffset, trackOffset):
         relative = absolute - counterStart
 
         currentTrack.index(i, path=s.path,
-            absolute=absolute,
-            relative=relative,
-            counter=c)
+                           absolute=absolute,
+                           relative=relative,
+                           counter=c)
         logger.debug(
             '[track %02d index %02d] trackOffset %r, added %r',
-                currentTrack.number, i, trackOffset,
-                currentTrack.getIndex(i))
-
+            currentTrack.number, i, trackOffset,
+            currentTrack.getIndex(i))
 
     def parse(self):
-        # these two objects start as None then get set as real objects,
-        # so no need to complain about them here
-        __pychecker__ = 'no-objattrs'
         currentFile = None
         currentTrack = None
 
         state = 'HEADER'
-        counter = 0 # counts sources for audio data; SILENCE/ZERO/FILE
+        counter = 0  # counts sources for audio data; SILENCE/ZERO/FILE
         trackNumber = 0
         indexNumber = 0
-        absoluteOffset = 0 # running absolute offset of where each track starts
-        relativeOffset = 0 # running relative offset, relative to counter src
-        currentLength = 0 # accrued during TRACK record parsing;
-                          # length of current track as parsed so far;
-                          # reset on each TRACK statement
-        totalLength = 0 # accrued during TRACK record parsing, total disc
-        pregapLength = 0 # length of the pre-gap, current track in for loop
+        absoluteOffset = 0  # running absolute offset: where each track starts
+        relativeOffset = 0  # running relative offset, relative to counter src
+        # currentLength is accrued during TRACK record parsing length
+        # of current track as parsed so far reset on each TRACK statement
+        currentLength = 0
+        totalLength = 0  # accrued during TRACK record parsing, total disc
+        pregapLength = 0  # length of the pre-gap, current track in for loop
```

Comment:
I also still think it would make more sense to put the comments before the assignments here, esp. considering the long comment for `relativeOffset`.